multiqc/modules/bamqc/assets/js/multiqc_bamqc.js

```diff
@@ -15,9 +15,21 @@ $(function () {
                 add_warning = true;
             }
         });
+        var pass_percent = (v['pass']/total)*100;
+        var fail_percent = (v['fail']/total)*100;
+        var pass_number_of_digits = v['pass'].toString().length;
+        var fail_number_of_digits = v['fail'].toString().length;
+        // Each digit needs around 8px of space
+        if (pass_percent != 0 && pass_percent < pass_number_of_digits * 8) {
+            pass_percent = pass_number_of_digits * 8;
```

Comment:
It looks like `pass_percent` was intialized as a percentage (line 18) but you're reassigning it here to be a number of pixels, which is not intuitive. Please save `pass_number_of_digits * 8` to a new variable and use that below.